### DISCUSSION TITLE
Github pages website linking to ".md" file instead of ".html" file as expected

### DISCUSSION THREAD

**Discussion by sohkamyung** [community/community#34789] [Other Feature Feedback, Questions, & Ideas] [answered]
My github pages site: https://github.com/sohkamyung/sohkamyung.github.io and the website displayed by it: https://sohkamyung.github.io/

I'm experimenting with setting up a website using github pages. My initial links for testing are as follows:

In markdown on github site, I have set up this structure:

/README.md ---> /reviews/index.md ---> /reviews/nonfiction/index.md

My expectation is that on the markdown (.md) files get generated into .html files. This works for the top level README.md file, but not for the next index.md file:

/index.html ---> /reviews/index.html ---> /reviews/nonfiction/index.md (I expected this to be index.html).

What am I doing wrong? Is it because github pages is only limited to one level of subdirectory?

**Comment by radar-blip** [comment]
You dont have an index.html in your repository. 
If Github Pages can't find an index.html, it will use the README.md. That's whats happening in your case.
You just need to create an index.html

**Comment by dpuga1** [comment]
I’m new to this so I apologize in advance.
How do I fix that problem?

**Comment by jfullstackdev** [comment, marked as answer]
without `index.html`, the default README is the one to be rendered, but you both have README and `index.md`, which is another option for a default page, and I'm seeing, you're mixing it, sometimes you use README, sometimes you want to use the `index.md` as the landing page, because it's the one being rerouted to

you might want to be consistent with that, if it's only README, all of the subpages will only contain README, and you will prevent your confusion, you don't need to include at the URL the filename, as they are default once you visited the folder, it will be rendered as expected

**Reply by sohkamyung** [reply]
Thank you for the reply. I have updated my github pages to remove the index.md files and only use README.md as my default page.

Things are now working as expected.

**Reply by GruntProof** [reply]
Thanks jdevstatic. Worked for me!
